insights/parsers/findmnt.py

```diff
@@ -0,0 +1,79 @@
+        /proc                                                           proc                                  proc            rw,nosuid,nodev,noexec,relatime                                               shared
+        /dev                                                            devtmpfs                              devtmpfs        rw,nosuid,seclabel,size=8035516k,nr_inodes=2008879,mode=755                   shared
+        /sys/kernel/security                                            securityfs                            securityfs      rw,nosuid,nodev,noexec,relatime                                               shared
+        /dev/shm                                                        tmpfs                                 tmpfs           rw,nosuid,nodev,seclabel                                                      shared
+        /run/netns                                                      tmpfs[/netns]                         tmpfs           rw,nosuid,nodev,seclabel,mode=755                                             shared
+        /netns                                                          tmpfs[/netns]                         tmpfs           rw,nosuid,nodev,seclabel,mode=755                                             shared
+        /run/netns/qdhcp-08f32dab-927e-4a61-933d-57d425827b57           proc                                  proc            rw,nosuid,nodev,noexec,relatime                                               shared
+        /run/netns/qdhcp-fd138c0a-5ec7-44f8-88df-0501c4c7a968           proc                                  proc            rw,nosuid,nodev,noexec,relatime                                               shared
+
+
+    Examples:
+
+        >>> output.search_target('shm') == [{'target': '/dev/shm', 'source': 'tmpfs', 'fstype': 'tmpfs', 'options': 'rw,nosuid,nodev,seclabel', 'propagation': 'shared'}]
+        True
+        >>> len(output.target_startswith('/run/netns')) == 3
+        True
+        >>> output.target_startswith('/run/netns')[0].get('propagation', None) == 'shared'
+        True
+
+    Attributes:
+        cols (list): List of key value pair derived from the command.
+
+        keywords (list): keywords present in the command, each
+                        keyword is converted to lowercase.
+    """
+    def parse_content(self, content):
+        self.cols = []
+        self.keywords = []
+        if not content:
+            return
+
+        self.cols = parse_fixed_table(content,
+                                      header_substitute=[
+                                          ('TARGET', 'target'),
+                                          ('SOURCE', 'source'),
+                                          ('FSTYPE', 'fstype'),
+                                          ('OPTIONS', 'options'),
+                                          ('PROPAGATION', 'propagation')])
+        self.keywords = [col['target'].lower() for col in self.cols]
```

Comment:
If you convert the `target` to lowercases on purpose here? But I think there are some risks to do so:

1. In most Linux systems, the file path is case sensitive, which means the `/mnt/Abc` is a different directory with `/mnt/abc`. By converting them to lowercase, it's not possible to distinguish them when they are both presented in the output.
2. You just converted them in the `self.keywords`, but didn't in `self.cols`, which leads to **data inconsistent**. E.g.
If we have output:
~~~
/run/netns/Qdhcp-08f32dab-927e-4a61-933d-57d425827b57           proc                                  proc            rw,nosuid,nodev,noexec,relatime
/run/netns/qdhcp-fd138c0a-5ec7-44f8-88df-0501c4c7a968           proc                                  proc            rw,nosuid,nodev,noexec,relatime
~~~
The value of `self.keywords` is `[
'/run/netns/qdhcp-08f32dab-927e-4a61-933d-57d425827b57', '/run/netns/qdhcp-fd138c0a-5ec7-44f8-88df-0501c4c7a968']`
but in the `self.cols`, it's not converted to lowercase. So, only one search result can be found by  `search_target('qdhcp')` : `[{'target': '/run/netns/qdhcp-fd138c0a-5ec7-44f8-88df-0501c4c7a968', 'source': 'proc', 'fstype': 'proc', 'options': 'rw,nosuid,nodev,noexec,relatime', 'propagation': 'private'}]`

Reply:
The purpose was to convert to lower case for the ease of comparison. Besides I did not find the mount points in upper cases(specially in `findmnt` o/p on my system, OSP env. or in the examples mentioned in the https://github.com/RedHatInsights/insights-core/issues/2368). Also I think the uuid will not be in upper case(at-least OSP environment).

Finally I've removed the conversion to lower case as suggested.

Reply:
That's good. If there are no uppercase mount points, then this `lower` is even more unnecessary, besides, it's incorrect.

Reply:
Thanks for pointing out.